mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -877,13 +882,15 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
     private fun showSnackbar(
         @StringRes messageResId: Int,
         @StringRes actionResId: Int = 0,
+        tag: String? = null,
         onClickListener: (() -> Unit)? = null
     ) {
         hideSnackbar()
         val snackbar = Snackbar.make(findViewById(android.R.id.content), messageResId, Snackbar.LENGTH_LONG)
         if (actionResId != 0 && onClickListener != null) {
             snackbar.setAction(actionResId) { onClickListener() }
         }
+        snackbar.view.tag = tag.toString()
```

Comment:
`toString()` seems useless on a string ;-)

Reply:
Yes, but not on a `String?`.

Reply:
But you don't need to convert to the string 'null' anyway. If a null tag is passed, why shouldn't we assign it as-is? I think we could even make the `tag` parameter to be of type `Any?`.

Reply:
I want to keep tag a `String?` as long we don't need an `Any?` tag.

Reply:
That's fine, just the toString call is useless ;-)